Example/InstanceID/Tests/FIRInstanceIDKeyPairMigrationTest.m

```diff
@@ -156,7 +156,7 @@ - (void)testMigrationIfLegacyKeyPairsExist {
 }
 
 - (void)testUpdateKeyRefWithTagRetainsAndReleasesKeyRef {
-#if TARGET_OS_IOS || TARGET_OS_TV
+#if (TARGET_OS_IOS || TARGET_OS_TV) && !defined(TARGET_OS_MACCATALYST)
```

Comment:
I think, we can add the condition to `generateKeyRef` method as well because it is the one failing.

Reply:
is it just test falling? I wonder if IID/FIS function normally in catalyst.

Reply:
Yes - since TARGET_OS_IOS is set, but it's macOS.